remsfal-service/src/test/java/de/remsfal/service/boundary/ProjectResourceTest.java

```diff
@@ -32,60 +30,60 @@ protected void setupTestUsers() {
     @Test
     void getProjects_FAILED_noAuthentication() {
         given()
-            .when()
-            .get(BASE_PATH)
-            .then()
-            .statusCode(Status.UNAUTHORIZED.getStatusCode());
+                .when()
```

Comment:
Please remove the unnecessary indentation. I added a rule that will check this in the future. As you can see in this PR https://github.com/remsfal/remsfal-backend/pull/180

We will also have a formatter which is currently in work in this PR https://github.com/remsfal/remsfal-backend/pull/166